migration/src/m20220101_000001_init.rs

```diff
@@ -124,6 +124,52 @@ impl MigrationTrait for Migration {
             )
             .await?;
 
+        manager
+            .create_table(
+                Table::create()
+                    .table(TokenOwnership::Table)
+                    .if_not_exists()
+                    .col(
+                        ColumnDef::new(TokenOwnership::Id)
+                            .big_integer()
+                            .not_null()
+                            .auto_increment()
+                            .primary_key(),
+                    )
+                    .col(ColumnDef::new(TokenOwnership::Owner).binary().not_null())
+                    .col(ColumnDef::new(TokenOwnership::Mint).binary().not_null())
+                    // TODO: Change this to a u64 here to avoid balance overflow.
+                    .col(
+                        ColumnDef::new(TokenOwnership::Amount)
+                            .big_integer()
+                            .not_null(),
+                    )
+                    .col(ColumnDef::new(TokenOwnership::Delegate).binary())
+                    .col(ColumnDef::new(TokenOwnership::Frozen).boolean().not_null())
+                    // TODO: Change this to a u64 here to avoid balance overflow.
+                    .col(ColumnDef::new(TokenOwnership::IsNative).big_integer())
```

Comment:
What is "IsNative" and why is it a bigint and not a bool?

Reply:
See question in Telegram. 

I am seeing that we have this field in the token indexing data:
    /// If is_native.is_some, this is a native token, and the value logs the rent-exempt reserve. An
    /// Account is required to be rent-exempt, so the value is used by the Processor to ensure that
    /// wrapped SOL accounts do not drop below this threshold.
    pub is_native: COption<u64>,
Does it make sense to have it as a COption<u64> in compressed accounts since we don’t have a notion of rent in compressed accounts?